docs-src/content/functions/crypto.yml

```
@@ -158,6 +158,57 @@ funcs:
         +hIz6+EUt/Db51awO7iCuRly5L4TZ+CnMAsIbtUOqsqwSQDtv0AclAuogmCst75o
         aztsmrD79OXXnhUlURI=
         -----END PUBLIC KEY-----
+  - name: crypto.Ed25519GenerateKey
+    experimental: true
+    released: v4.0.0
```

these need to be commented out (like in my original suggestion) beause this is not yet released (I'll update these when the release happens):

```suggestion
    # released: v4.0.0
```

---

I have it resolved locally, and am waiting to push up the latest changes. I sent a question to you via email.